queue/queueservice.js

```diff
@@ -357,7 +494,7 @@ class QueueService {
   }
 
   /** Get signed URLs for polling and deleting from the azure queue */
-  async signedPendingPollUrl(provisionerId, workerType) {
+  async signedPendingPollUrls(provisionerId, workerType) {
     // Find name of azure queue
     var queueName = await this.ensurePendingQueue(provisionerId, workerType);
```

Comment:
Ok, I see earlier why returning just the queue name is easier here, but won't there be more than 1 pending queue?